docs/cloud-native-security/falco-setup.asciidoc

```diff
@@ -0,0 +1,58 @@
+
+[discrete]
+[[ingest-falco-setup-kibana]]
+== Configure {elastic-sec} to receive Falco data
+
+In {elastic-sec}:
+
+. Click **Add integrations**.
+. Search the Integrations page for `Falco`, then select it.
+. Go to the Falco integration's **Settings** tab. 
+. Click **Install Falco**, then confirm by clicking **Install Falco** again. Installation should take less than a minute. 
+
+image::images/falco-kibana-setup.gif[A gif showing the steps to set up Falco in Kibana]
+
+{elastic-sec} is now ready to receive data from Falco. The Falco integration page now has an **Assets** tab where you can inspect the newly installed assets that help to ingest Falco data.
+
+Next, to make alerts from Falco appear on {kib}'s Alerts page:
+
+. Find the **Rules** page in the main menu or by using the {kibana-ref}/introduction.html#kibana-navigation-search[global search field].
+. Click **Detection rules (SIEM)**.
+. Search for a rule called `External Alerts`. Enable it using the **Enabled** toggle on the right.
+
+image::images/falco-external-alerts-rule.gif[A gif showing the steps to enable the External Alerts rule]
+
+[discrete]
+[[ingest-falco-setup-falco]]
+== Configure Falco and Falcosidekick
+
+Multiple methods for configuring Falco to send data to {es} are available. This guide uses the https://github.com/falcosecurity/falcosidekick/blob/master/docs/outputs/elasticsearch.md[Falco sidekick on Docker using environment variables] method. (Falcosidekick is a proxy forwarder which can send alerts from your Falco deployments to {es}.)
+
+[discrete]
+=== Configure Falco:
+
+. Refer to Falco's documentation to https://falco.org/docs/setup/packages/[install Falco on the Linux VMs you wish to monitor]. 
+. Once Falco is installed, update its configuration `.yaml` as follows:
+.. Enable JSON output: `json_output=TRUE`
+.. Enable HTTP output: `http_output=TRUE`
+.. In the `http_output` section, for the `url` value, enter the port where Falcosidekick will listen.
+
+[discrete]
+=== Configure Falcosidekick:
+
+. Refer to Falcosidekick's documentation to https://github.com/falcosecurity/falcosidekick?tab=readme-ov-file#installation[install Falcosidekick].
+. Use the https://github.com/falcosecurity/falcosidekick?tab=readme-ov-file#installation[Falcosidekick on Docker using environment variables] method and set your environment variables as follows:
+.. `ELASTICSEARCH_HOSTPORT`: your {es} endpoint URL, which can be found under **Connection details** on the upper right of the **Integrations** page in {kib}.
+.. `ELASTICSEARCH_INDEX`: the {es} index where you want to store Falco logs.
+.. `ELASTICSEARCH_SUFFIX`: The frequency with which you want the {es} index suffix to change. Either `daily`, `monthly`, `annually`, or `none`. 
+.. `ELASTICSEARCH_USERNAME` and `ELASTICSEARCH_PASSWORD`: the username and password for an account on your {es} instance, with at least the following permissions: Which permissions, Nick?
```

Comment:
> Which permissions, Nick?

For both username/password and API key, we should note that user should follow the principles of least privilege, with permissions limited to creating and writing to Falco indices. Maybe we could point user to [this](https://www.elastic.co/guide/en/kibana/current/api-keys.html) for API key and for username/password I assume there is a page similar to APIkey, which I couldn't locate. The closet one I could locate is [this](https://www.elastic.co/guide/en/elasticsearch/reference/current/setting-up-authentication.html)